### DISCUSSION TITLE
How to add new vite app to existing turbo monorepo?

### DISCUSSION THREAD

**Discussion by rickvian** [vercel/turborepo#8274] [Help] [answered]
### Summary

Hi, i already have existing turborepo project,
to add new vite-app, 

can i just run create-vite-app under app/ folder?

**Comment by versecafe** [comment]
@rickvian you can, just double check your package manager links up everything correctly, `pnpm` seems to always work fine but `bun` has been having issues with it

**Comment by anthonyshew** [comment, marked as answer]
We added a guide for this here: https://turbo.build/repo/docs/guides/frameworks/vite
